### DISCUSSION TITLE
Can CUDA 11.7 be used with Grok 1.0 instead of CUDA 12?

### DISCUSSION THREAD

**Discussion by bsab** [xai-org/grok-1#148] [Q&A]
Hello everyone, I have **16 Nvidia A100 40GB GPUs** available and I'm considering using them with Grok 1.0. I've read in the requirements that CUDA 12 is needed, but I was wondering if it's possible to use **CUDA 11.7** instead. Does anyone have any experience or information about this? Thank you in advance for your help.

**Comment by StavrosD** [comment]
CUDA 12 supports A100, there is no reason to use an older version.
https://developer.nvidia.com/blog/cuda-toolkit-12-0-released-for-general-availability/

**Reply by bsab** [reply]
Hey @StavrosD, unfortunately I don't have the privileges to upgrade it
